.github/workflows/pipeline.yml

```diff
@@ -4,12 +4,14 @@ on:
   push:
     branches: [ master, dev ]
 env:
-  tf_version: '0.12.24'
+  tf_version: '0.12.26'
 
 jobs:
   pipeline:
     name: Pipeline
     runs-on: ubuntu-latest
+    env:
+      TF_IN_AUTOMATION: 'true'
```

Comment:
See https://learn.hashicorp.com/terraform/development/running-terraform-in-automation#controlling-terraform-output-in-automation for info about this variable.